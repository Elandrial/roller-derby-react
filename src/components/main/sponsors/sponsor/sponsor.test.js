import Sponsor from './sponsor';
import renderer from 'react-test-renderer';

describe('Pager component',() => {
  it('renders correctly', () =>{
    const tree = renderer
    .create(<Sponsor imageUrl={"testimage.jpg"} name={"test"}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  })
});