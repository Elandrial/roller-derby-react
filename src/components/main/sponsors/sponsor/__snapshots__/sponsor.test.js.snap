// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pager component renders correctly 1`] = `
<div>
  <img
    alt="test"
    src="/images/testimage.jpg"
  />
</div>
`;
